OSDOCS-19410-4 back porting most recent zero-trust docs to 4.18 PR#4 (#110952)
openshift/openshift-docs: security/zero_trust_workload_identity_manager/zero-trust-manager-reconciliation.adoc

:_mod-docs-content-type: ASSEMBLY
[id="zero-trust-manager-reconciliation_{context}"]
= Enabling create-only mode for the Zero Trust Workload Identity Manager
include::_attributes/common-attributes.adoc[]
:context: zero-trust-manager-reconciliation

toc::[]

[role="_abstract"]
To pause Operator reconciliation, enable `create-only` mode by setting an environment variable in the subscription object. By setting this value, you can perform manual configurations or debug the Operator without the controller overwriting your changes.

The following scenarios are examples of when the `create-only` mode might be of use:

> **Manual Customization Required**: You need to customize operator-managed resources (ConfigMaps, Deployments, DaemonSets, etc.) with specific configurations that differ from the Operator's defaults

Reviewer: ```suggestion
**Manual Customization Required**: You need to customize operator-managed resources (ConfigMaps, Deployments, DaemonSets, and so on) with specific configurations that differ from the Operator defaults.
```
IBM doesn't allow the latin abbreviations; or possessives. And all of these bullets have complete sentences, so really should have periods (that's a quibble, do as you see fit).

Author: Done

> **Manual Customization Required**: You need to customize operator-managed resources, such as ConfigMaps, Deployments, DaemonSets, and so on, with specific configurations that differ from the Operator defaults.

Author: Don't need to change to Operators
